omod-2.0/src/test/java/org/openmrs/module/attachments/rest/AttachmentRestController2_0Test.java

```diff
@@ -94,6 +100,93 @@ public void shouldGetRefByUuid() {
 	public void shouldGetFullByUuid() {
 	}
 	
+	@Test
+	public void createAttachment_shouldCreateANewAttachment() throws Exception {
+		
+		SimpleObject attachment = new SimpleObject();
+		attachment.add("uuid", obs.getUuid());
+		attachment.add("comment", obs.getComment());
+		attachment.add("complexData", obs.getComplexData());
+		
+		String json = "{\"uuid\":\"" + getUuid() + "\",\"attachment\":\"" + attachment + "\"}";
+		
+		try {
+			json = new ObjectMapper().writeValueAsString(attachment);
+		}
+		catch (IOException e) {
+			// TODO Auto-generated catch block
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotprintout

Reply:
Hope using logs to catch exception would work here